LICENSE

```diff
@@ -1,13 +1,201 @@
-Copyright 2015 Technique Software
```

Comment:
It's interesting that the apache website seems to instructs you to keep it the way you had it before: https://www.apache.org/licenses/LICENSE-2.0. Seems like trusting githubs instruction and using the more verbose text can't hurt though.

Reply:
I think what you're looking for is this https://github.com/TechniqueSoftware/react-json-schema/pull/13/files/28a6d36d632a8aba6b5cf3b6d7f9cfafd02d8a04#diff-9879d6db96fd29134fc802214163b95aR189

Reply:
@eiof What I'm referring to is the Appendix at the bottom. It seems to indicate that you shouldn't include the actual terms and agreements but just the boilerplate which links to the larger text. This is what I'm expecting that we need to include: 

```
Copyright [yyyy] [name of copyright owner]

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
```

Reply:
Actually it seems like either way is fine. 

NYTimes has a help document instructing people to only use the boilerplate/short version https://github.com/NYTimes/license ...

... but then I see them using it the way you're proposing in the PR: https://github.com/NYTimes/gizmo/blob/master/LICENSE